api/envoy/extensions/filters/http/mutation/v3alpha/BUILD

```diff
@@ -0,0 +1,12 @@
+# DO NOT EDIT. This file is generated by tools/proto_format/proto_sync.py.
```

Comment:
+1, was going to suggest a name change. "offload" is probably fine though we should try to think of other names also. "offload_via_api" "offload_via_grpc" etc.

Reply:
I am going going to propose "external processing filter," or "ext_proc" for short. But let's get through the many comments on this PR because when I change the name I think GitHub will toss many of the comments. LMK if that name is not OK!